x-pack/plugins/telemetry_collection_xpack/schema/xpack_plugins.json

```diff
@@ -4050,6 +4050,35 @@
             },
             "deprecated": {
               "type": "long"
+            },
+            "app": {
+              "properties": {
+                "search": {
+                  "type": "long"
+                },
```

Comment:
@Bamieh that's a lot of additions, is that fine with you?

Reply:
Thanks for raising this @pgayvallet . The main reason why there are so many additions is that adding new export types multiplies what we track with telemetry:
   - PDF
   - PNG
   - CSV
   - PDFV2
   - PNGV2
   - CSVV2

For every export type, there is telemetry tracking for which apps were used, and what the layout options were.

This PR adds all of the V2 types to the schema

Reply:
I was on holidays this week. It is fine. better than the alternatives!